src/components/forms/PlanForm/index.tsx

```diff
@@ -163,6 +165,11 @@ const PlanForm = (props: PlanFormProps) => {
   const irsActivities = getFormActivities(IRSActivities);
   const fiActivities = getFormActivities(FIActivities);
 
+  let filteredFIReasons: FIReasonType[] = [...FIReasons];
+  if (LANGUAGE === 'th') {
+    filteredFIReasons = FIReasons.filter(reason => reason !== CASE_TRIGGERED);
+  }
+
```

Comment:
I would vote for using an env variable (something like enabled_fi_reasons) instead of relying on the language.  It may be possible that we have an English Thai deployment, for example.